authentication/tests/test_user_autologout.py

```diff
@@ -0,0 +1,72 @@
+from datetime import timedelta
+from rest_framework import status
+from rest_framework.test import APITestCase
+from rest_framework.authtoken.models import Token
+
+from authentication.models import CustomUser
+from .unittest_helper import AnyInt
+
+
+class UserLogoutAPITests(APITestCase):
+    def setUp(self):
+        self.test_user = CustomUser.objects.create_user(
+            person_email="test@test.com",
+            password="Test1234",
+            person_name="Test",
+            person_surname="Test",
+        )
+        self.test_user.is_active = True
```

Comment:
Is there a reason for not just adding it to `create_user`?

Reply:
Added to create_user in all the authentication tests